gcp/bootstrap.tf

```diff
@@ -0,0 +1,260 @@
+    dcos_exhibitor_explicit_keys = "${var.dcos_exhibitor_explicit_keys}"
+    dcos_exhibitor_storage_backend = "${var.dcos_exhibitor_storage_backend}"
+    dcos_exhibitor_zk_hosts = "${var.dcos_exhibitor_zk_hosts}"
+    dcos_exhibitor_zk_path = "${var.dcos_exhibitor_zk_path}"
+    dcos_gc_delay = "${var.dcos_gc_delay}"
+    dcos_http_proxy = "${var.dcos_http_proxy}"
+    dcos_https_proxy = "${var.dcos_https_proxy}"
+    dcos_log_directory = "${var.dcos_log_directory}"
+    dcos_master_discovery = "${var.dcos_master_discovery}"
+    dcos_master_dns_bindall = "${var.dcos_master_dns_bindall}"
+    # TODO(bernadinm) Terraform Bug: 9488.  Templates will not accept list, but only strings.
+    # Workaround is to flatten the list as a string below. Fix when this is closed.
+    dcos_master_list = "\n - ${join("\n - ", google_compute_instance.master.*.network_interface.0.address)}"
+    dcos_no_proxy = "${var.dcos_no_proxy}"
+    dcos_num_masters = "${var.num_of_masters}"
+    dcos_oauth_enabled = "${var.dcos_oauth_enabled}"
+    dcos_overlay_config_attempts = "${var.dcos_overlay_config_attempts}"
+    dcos_overlay_enable = "${var.dcos_overlay_enable}"
+    dcos_overlay_mtu = "${var.dcos_overlay_mtu}"
+    dcos_overlay_network = "${var.dcos_overlay_network}"
+    dcos_process_timeout = "${var.dcos_process_timeout}"
+    dcos_previous_version = "${var.dcos_previous_version}"
+    #  ------ TODO(bernadinm) uncomment this line below
+    # dcos_agent_list = "\n - ${join("\n - ", google_compute_instance.agent.*.network_interface.0.address)}"
+    # TODO(bernadinm) Terraform Bug: 9488.  Templates will not accept list, but only strings.
+    # Workaround is to flatten the list as a string below. Fix when this is closed.
+    dcos_resolvers  = "\n - ${join("\n - ", var.dcos_resolvers)}"
+    dcos_rexray_config_filename = "${var.dcos_rexray_config_filename}"
+    dcos_rexray_config_method = "${var.dcos_rexray_config_method}"
+    dcos_s3_bucket = "${var.dcos_s3_bucket}"
+    dcos_s3_prefix = "${var.dcos_s3_prefix}"
+    dcos_security  = "${var.dcos_security}"
+    dcos_superuser_password_hash = "${var.dcos_superuser_password_hash}"
+    dcos_superuser_username = "${var.dcos_superuser_username}"
+    dcos_telemetry_enabled = "${var.dcos_telemetry_enabled}"
+    dcos_use_proxy = "${var.dcos_use_proxy}"
+    dcos_zk_agent_credentials = "${var.dcos_zk_agent_credentials}"
+    dcos_zk_master_credentials = "${var.dcos_zk_master_credentials}"
+    dcos_zk_super_credentials = "${var.dcos_zk_super_credentials}"
+    dcos_cluster_docker_registry_url = "${var.dcos_cluster_docker_registry_url}"
+    dcos_rexray_config = "${var.dcos_rexray_config}"
+    dcos_ip_detect_public_contents = "${var.dcos_ip_detect_public_contents}"
+    dcos_enable_docker_gc = "${var.dcos_enable_docker_gc}"
+    dcos_staged_package_storage_uri = "${var.dcos_staged_package_storage_uri}"
+    dcos_package_storage_uri = "${var.dcos_package_storage_uri}"
+}
```

Comment:
i don't see the fault domain option here, should it be added?

Reply:
Yeah, it will land it 1.11 so it doesn't exist yet  but I will add it now since I know it's coming.